charts/reform-scan-blob-router/values.yaml

```diff
@@ -19,6 +19,12 @@ java:
         - crime-storage-account-primary-key
         - flyway-password
   environment:
+    DB_HOST: blob_router-{{ .Values.global.environment }}.postgres.database.azure.com
+    DB_PORT: '5432'
+    DB_NAME: blob_router
+    DB_USER: blob_router@blob_router-{{ .Values.global.environment }}
+    DB_CONN_OPTIONS: ?sslmode=require
+
```

Comment:
does this change need to bump the chart version?

Reply:
oops, I've updated it, but then merged new master with bumped version. Need to bump again

Reply:
done